packages/sdk/utils/src/sign-typed-data-utils.ts

```diff
@@ -40,14 +40,17 @@ export interface EIP712TypedData {
 
 /** Array of all EIP-712 atomic type names. */
 export const EIP712_ATOMIC_TYPES = [
-  'bytes1',
-  'bytes32',
-  'uint8',
-  'uint64',
-  'uint256',
-  // This list should technically include all types from uint8 to uint256, and int8 to int256
-  'int8',
-  'int256',
+  // bytes types from 1 to 32 bytes
+  // and uint/int types from 8 to 256 bits
+  ...(() => {
+    const result = []
+    for (let i = 1; i <= 32; i++) {
+      result.push('bytes' + i)
+      result.push('uint' + i * 8)
+      result.push('int' + i * 8)
+    }
+    return result
+  })(),
```

Comment:
This array is now bigger, but hopefully it shouldn't be a performance problem for finding values.

Reply:
Good idea about putting more common types first 👍 
I did a naive take at this in https://github.com/celo-org/celo-monorepo/pull/10224/commits/4bcde56c434b7ce57546405a9c005e8d1372e3c2